.php-cs-fixer.dist.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+$config = \TYPO3\CodingStandards\CsFixerConfig::create();
+$config
+    ->getFinder()->in(__DIR__)
+    ->exclude([
+        'Documentation',
```

Comment:
What is the difference of the both directories? "Documentation" is not available yet.

Reply:
I always put the stuff I want to test into Documentation

Reply:
Then, should be the stuff from "Documentation" be code-styled?

And what is the purpose of "docs"? Actually, there is only one rst file. Should it be delivered to the users?

Reply:
The docs was just added by @jaapio and explains how you can use the mono-repository. So I guess it should be delivered to the user if the complete reposiory is delivered to them?

Reply:
And no, I wouldn't want an external script to code-style my test Data as it comes from a different source, might have different PHP versions etc

Reply:
Aaaah, it is an "exclude", sorry, just looked at the folder names....